integration/docker/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN cd /alluxio-csi && \
 #       - https://github.com/moby/moby/issues/6119
 # ADD with chown doesn't chown the files inside tarball
 #   See - https://github.com/moby/moby/issues/35525
-FROM alpine:3.10.2 AS alluxio-extractor
+FROM alpine:3.17 AS alluxio-extractor
```

Comment:
we cannot take in this alpine version change without extensive testing. i would advise to omit it from this PR

Reply:
version 1.18 is the most stable version that has undergone a lot of testing in many projects

Reply:
this is referring to the alpine version, not the golang version.

Reply:
alpine:3.17 is currently the most stable version with no cve vulnerabilities